src/api.rs

```diff
@@ -109,9 +108,10 @@ impl From<u64> for AnchoringChainLength {
     }
 }
 
-/// Public API part of the Exonum Bitcoin anchoring service.
+/// Public API client for the Exonum Bitcoin anchoring service.
+#[async_trait(?Send)]
```

Comment:
What's the reason for the `?Send` qualifier? It would make it impossible to spawn a future returned by an implementation (or any future `await`ing it) on the `tokio` runtime, for example.

Reply:
Because without `?Send` qualifier I get an error.

```
future cannot be sent between threads safely
within `exonum_testkit::api::TestKitApi`, the trait `std::marker::Sync` is not implemented for `std::rc::Rc<awc::ClientConfig>`
required for the cast to the object type `dyn std::future::Future<Output = std::result::Result<exonum_crypto::Hash, exonum_api::error::Error>> + std::marker::Send`
mod.rs(431, 9): future is not `Send` as this value is used across an await
future cannot be sent between threads safely
the trait `std::marker::Send` is not implemented for `dyn std::ops::FnOnce(reqwest::async_impl::request::RequestBuilder) -> reqwest::async_impl::request::RequestBuilder`
```